pkg/config/patch_test.go

```diff
@@ -195,9 +204,10 @@ metadata:
 		name: "add labels to unpatchedDeployment without affecting existing labels (JSON6902, json)",
 		object: applyappsv1.Deployment("test", "test").
 			WithLabels(map[string]string{"initial": "label"}),
+		out: applyappsv1.Deployment("test", "test"),
 		patches: []v1alpha1.Patch{{
 			Kind:  "Deployment",
-			Patch: json.RawMessage(`{"op": "add", "path": "/metadata/labels", "value":{"added":"via-patch"}}`),
+			Patch: json.RawMessage(`{"op": "add", "path": "/metadata/labels/added", "value":"via-patch"}`),
```

Comment:
the previous patch replaces all of the labels - the test case shouldn't've had both the old and new labels